tests/wallet/cat_wallet/config.py

```diff
@@ -1,5 +1,4 @@
 from __future__ import annotations
 
-parallel = False
```

Comment:
Do you remember why you set it to `False` when you introduced it?

Reply:
FYI, we did run a 25x set of flake check runs on the edited test directories (admittedly, fixture changes could affect more than those) and got a single timeout that was 3x out from normal runtimes and a single test failure.  Just evidence that we aren't making things catastrophically worse in terms of flakes, at least.

Reply:
@xdustinface I set it to false when I changed the default to be `true`, in order to preserve the existing behavior